SymHop/src/SymHop.cpp

```diff
@@ -1279,6 +1279,203 @@ QString Expression::toString() const
     return ret;
 }
 
+QString Expression::toLaTeX() const
+{
+    QString ret;
+
+    if(this->isSymbol())
+    {
+        QStringList greekSymbols = {"alpha","beta","gamma","Gamma","delta","Delta","epsilon","varepsilon","zeta","eta","theta","vartheta","Theta","iota","kappa","lambda","Lambda","mu","nu","xi","Xi","pi","Pi","rho","varrho","sigma","Sigma","tau","upsilon","Upsilon","phi","varphi","Phi","chi","psi","Psi","omega","Omega"};
```

Comment:
Some of these have only lower case version, maybe intended, I dont know

Reply:
Yes, it is intentional. Many upper case greek letters are equivalent to latin letters, and those don't have latex commands.